hexrd/ui/calibration_slider_widget.py

```diff
@@ -290,3 +293,12 @@ def emit_update_if_polar(self):
     def reset_config(self):
         HexrdConfig().restore_instrument_config_backup()
         self.update_gui_from_config()
+
+    def update_labels(self):
+        if HexrdConfig().euler_angle_convention is None:
+            return
```

Comment:
We might still want to set them to `XYZ` here, because if the user switches to "ZXZ", and then switches "None", it will still display "ZXZ".

Reply:
Fixed